controller/doc_manager.mli

```diff
@@ -19,12 +19,12 @@ module Check : sig
   (** Check pending documents, return [None] if there is none pending, or
       [Some rqs] the list of requests ready to execute after the check. Sends
       progress and diagnostics notifications to [ofmt]. *)
```

Comment:
Need to update comment. Also how am I supposed to read `ofn`? "output format...?"

Reply:
Thanks, comment updated. `ofn` means "output function"